inbm/dispatcher-agent/tests/unit/source/test_source_command.py

```diff
@@ -227,7 +227,7 @@ def test_do_source_command_add(
             "dispatcher.source.source_command.create_application_source_manager",
             OsType.Ubuntu,
             ApplicationUpdateSourceParameters(
-                source_list_file_name="filename", sources=["source1", "source2"]
+                sources=["source1", "source2"], source_list_file_name="filename"
```

Comment:
**Suggestion:** Ensure consistent ordering of parameters when creating instances of ApplicationUpdateSourceParameters for readability and maintainability. [maintainability]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
            ApplicationUpdateSourceParameters(
                source_list_file_name="filename",
                sources=["source1", "source2"]
            ),
```